internal/ui/settings.go

```diff
@@ -179,7 +179,7 @@ func (d *deskSettings) load() {
 		d.launcherZoomScale = 2.0
 	}
 
-	moduleNames := fyne.CurrentApp().Preferences().StringWithFallback("modulenames", "Battery|Brightness|Sound")
+	moduleNames := fyne.CurrentApp().Preferences().StringWithFallback("modulenames", "Battery|Brightness|Sound|Launch Calculations")
```

Comment:
Why is the calculator launch module the only one loaded here?  Why not URL?

Also - I suppose it would be trivial to add an open in terminal with paths module?

Reply:
It seemed like calculator was clearly beneficial but URLs perhaps less so.
On reflection maybe they should all be in a single "Launch Suggestions" module and just remove the user ability to turn them on/off (until we add module config)?